pyartcd/pyartcd/pipelines/promote.py

```diff
@@ -259,6 +259,10 @@ async def run(self):
                 verify_flaws = True
                 if "prerelease" in impetus_advisories.keys() or assembly_type == assembly.AssemblyTypes.PREVIEW:
                     verify_flaws = False
+
+                if assembly_type in [assembly.AssemblyTypes.PREVIEW, assembly.AssemblyTypes.CANDIDATE] and self.assembly.endswith(".0"):
```

Comment:
or self.assembly.endswith(".0") ?

Reply:
dang it, today's not my day.

fixed, repushed